docs/io/misc.rst

```diff
@@ -1,5 +1,5 @@
 **************************************************************
-Miscellaneous: HDF5, YAML, Parquet, pickle (`astropy.io.misc`)
+HDF5, Parquet, pickle, PyArrow CSV, YAML (`astropy.io.misc`)
```

Comment:
I just checked and nowhere in the links in this document is `pickle` used - I'd suggest to just remove it here while you are at it.

Reply:
OK, I see what happened. There are some pickle functions but they have been deprecated so the API docs got taken away. Good catch, I have taken `pickle` out of the title.